configurations/test_pipeline_configuration.py

```diff
@@ -92,22 +92,59 @@
             write_mode=WriteModes.CONCATENATE_TEXTS
         )
     ),
-    analysis_config=[
-        AnalysisDatasetConfiguration(
-            engagement_db_datasets = ["s01e01"],
-            dataset_type = DatasetTypes.RESEARCH_QUESTION_ANSWER
-        ),
-        AnalysisDatasetConfiguration(
-            engagement_db_datasets = ["gender"],
-            dataset_type = DatasetTypes.DEMOGRAPHIC
-        ),
-        AnalysisDatasetConfiguration(
-            engagement_db_datasets = ["location"],
-            dataset_type = DatasetTypes.DEMOGRAPHIC
-        ),
-        AnalysisDatasetConfiguration(
-            engagement_db_datasets = ["age"],
-            dataset_type = DatasetTypes.DEMOGRAPHIC
-        )
-    ]
+    analysis_configs=AnalysisConfigurations(
+        dataset_configurations=[
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["s01e01"],
+                dataset_type=DatasetTypes.RESEARCH_QUESTION_ANSWER,
+                raw_field="s01e01_raw",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("s01e01"),
+                        analysis_file_key="s01e01"
```

Comment:
Does it work to call this "analysis_dataset" instead of analysis_file_key?

Reply:
updated thanks.